blog/source/_posts/MIDI-velocity-transformer.md

```diff
@@ -0,0 +1,266 @@
+---
+title: MIDI Velocity Prediction with Transformer
```

Comment:
```suggestion
title: Modelling dynamic expression in piano performance
```

I think we shouldn't mention MIDI or velocity in the title, as it's to technical - my suggestions aims to provide broader goal for what we're doing. I think you can add "with Transformers" if you like